theme/src/js/helpers/gated-content.js

```diff
@@ -0,0 +1,49 @@
+export function gateArticle() {
+	const additionalInformationNode = document.getElementById(
+		'additionalInformation'
+	);
+
+	if (additionalInformationNode) {
+		additionalInformationNode.parentNode.removeChild(
+			additionalInformationNode
+		);
+	}
```

Comment:
Not sure if we care about IE, but if not, we can just use the `remove()` method.  See https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove.

We can also just polyfill the method.  Not sure if we are doing that in ZenDesk anywhere either.

Reply:
The code was rewritten to accommodate IE, unfortunately. I try to avoid polyfill anything to keep the file size down, since Zendesk has a hard limit set at below 300kb.

Reply:
Ok sounds good.